src/dataframe/dataframe.jl

```diff
@@ -952,28 +952,15 @@ Base.convert(::Type{DataFrame}, d::AbstractDict) = DataFrame(d)
 ##
 ##############################################################################
 
-function Base.push!(df::DataFrame, associative::AbstractDict{Symbol,Any})
-    i = 1
-    for nm in _names(df)
-        try
-            push!(df[nm], associative[nm])
-        catch
-            #clean up partial row
-            for j in 1:(i - 1)
-                pop!(df[_names(df)[j]])
-            end
-            msg = "Error adding value to column :$nm."
-            throw(ArgumentError(msg))
-        end
-        i += 1
-    end
-end
-
 function Base.push!(df::DataFrame, associative::AbstractDict)
     i = 1
     for nm in _names(df)
         try
-            val = get(() -> associative[string(nm)], associative, nm)
+            val = get(() -> (Base.depwarn("push!(::DataFrame, ::AbstractDict) with"*
```

Comment:
Should add a space at the end of the string, otherwise with the concatenation you'll get `withAbstractDict`.

Reply:
fixed